Makefile

```
@@ -0,0 +1,5 @@
+# Code style; defines `style-check` and `style-fix`.
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)